meshpy/four_c/dbc_monitor.py

```diff
@@ -91,7 +93,7 @@ def read_dbc_monitor_file(file_path):
 def dbc_monitor_to_input(input_file, file_path, step=-1, function=1, n_dof=3):
     """
     Convert the Dirichlet boundary condition monitor log to a Neumann
-    boundary condition input section.
+    boundary condition input section. Uses only the last force value.
```

Comment:
Should I remove this function?

Reply:
Yes, I think we should aim at having a “full” function that can also handle this case.